site/content/docs/5.0/examples/offcanvas/offcanvas.css

```diff
@@ -1,67 +1,3 @@
-html,
-body {
-  overflow-x: hidden; /* Prevent scroll on narrow devices */
-}
-
 body {
```

Comment:
Would be nice if we could replace this with a core util and get rid of the CSS file here. Maybe `pt-5` is enough @mdo?